web/models/meeting.ex

```diff
@@ -8,32 +9,43 @@ defmodule Meetings.Meeting do
   
   def new(data) do
     %Meetings.Meeting{ 
```

Comment:
using `%__MODULE__{..}` here instead of the explicit module name is better, in case the module name ever changes.